tests/check_ipc.c

```diff
@@ -429,15 +428,18 @@ run_ipc_server(void)
 		qb_ipcs_enforce_buffer_size(s1, max_size);
 	}
 	qb_ipcs_poll_handlers_set(s1, &ph);
+	qb_loop_signal_add(my_loop, QB_LOOP_HIGH, SIGTERM,
+			   NULL, exit_handler, &handle);
```

Comment:
Good catch that the former location made the very first invocation
of `qb_loop_signal_add` silently fail.

Can we have this fix as a separate commit, please?